packages/metrics-lwt/metrics-lwt.0.1.0/opam

```diff
@@ -14,8 +14,8 @@ build: [
 ]
 
 depends: [
-  "dune"
-  "metrics"
+  "dune" {>= "1.4"}
+  "metrics" {= version}
```

Comment:
I’m a bit thorn on this:
- on one hand this makes a lot of sense to keep it synched with metrics
- but on the other, they currently compile fine even with metrics 0.4.0 and i wouldn’t want to break existing users (e.g. lock files with both metrics-lwt.0.1.0 and metrics.0.2.0)

In practice i don’t think it matters too much for this one but i still feel conflicted.

Reply:
Should we just do 
```suggestion
  "metrics" {< "0.5"}
```
Then we would not break the current compatibility but ensure the syncing from 0.5 onwards